src/utils/Api.js

```diff
@@ -0,0 +1,79 @@
+class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  _checkResponse(res) {
+    return res.ok ? res.json() : Promise.reject(`Error: ${res.status}`);
+  }
+
+  getAppInfo() {
+    // call getUserInfo in this array
+    return Promise.all([this.getInitialCards(), this.getUserInfo()]);
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: this._headers,
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
+      return Promise.reject(`Error: ${res.status}`);
+    });
```

Comment:
Please, reuse `.then(this._checkResponse);` everywhere, to avoid code duplication